cloudify_marketplace/provision/security_reconfiguration_setup

```diff
@@ -1,17 +1,11 @@
 #! /usr/bin/env bash
 set -e
 
-if [[ "${CLOUDIFY_MANAGER_SECURITY_ENABLED}" == "true" ]]; then
-  # Security enabled, activate the cron job
-  sudo mv /tmp/cloudify_security_reconfiguration /opt/cloudify_security_reconfiguration
-  sudo chown root. /opt/cloudify_security_reconfiguration
-  sudo chmod 500 /opt/cloudify_security_reconfiguration
+# Security enabled, activate the cron job
+sudo mv /tmp/cloudify_security_reconfiguration /opt/cloudify_security_reconfiguration
```

Comment:
Oh... unless that's dealing with the authentication as well as the encryption... urgh.